src/renderer/components/password-dialog/password-dialog.js

```diff
@@ -0,0 +1,66 @@
+import Vue from 'vue'
+import { mapActions } from 'vuex'
+import FtSettingsSection from '../ft-settings-section/ft-settings-section.vue'
+import FtSelect from '../ft-select/ft-select.vue'
+import FtCard from '../ft-card/ft-card.vue'
+import FtInput from '../ft-input/ft-input.vue'
+import FtToggleSwitch from '../ft-toggle-switch/ft-toggle-switch.vue'
+import FtFlexBox from '../ft-flex-box/ft-flex-box.vue'
+import FtButton from '../ft-button/ft-button.vue'
+import FtPrompt from '../ft-prompt/ft-prompt.vue'
+
+export default Vue.extend({
+  name: 'PasswordDialog',
+  components: {
+    'ft-settings-section': FtSettingsSection,
+    'ft-select': FtSelect,
+    'ft-input': FtInput,
+    'ft-card': FtCard,
+    'ft-toggle-switch': FtToggleSwitch,
+    'ft-flex-box': FtFlexBox,
+    'ft-button': FtButton,
+    'ft-prompt': FtPrompt,
```

Comment:
Could you please remove the unused component imports, you only need to import and register the components that you use in the associated .vue file.